src/OrbitGgp/Client.cpp

```diff
@@ -35,12 +33,11 @@ void RunProcessWithTimeout(const QString& program, const QStringList& arguments,
 
   QObject::connect(timeout_timer, &QTimer::timeout, parent,
                    [process, timeout_timer, timeout, callback]() {
-                     if (process && !process->waitForFinished(kAdditionalSynchronousTimeoutInMs)) {
+                     if (process && !process->waitForFinished(0)) {
```

Comment:
What does `0` as the argument mean in that regard? I guess it will basically tell you whether the process is still running or not. In that case I would prefer to use `QProcess::state` instead.

If you meant to wait forever you have to pass in `-1` according to the docs.

Reply:
I used QProcessState now